lucene/core/src/test/org/apache/lucene/util/automaton/TestRegExp.java

```diff
@@ -88,10 +88,14 @@ public void testRepeatWithEmptyLanguage() throws Exception {
     assertTrue(a.toString().length() > 0);
   }
   
+  
+  boolean caseSensitiveQuery = true;
+  
   public void testCoreJavaParity() {
     // Generate random doc values and random regular expressions
     // and check for same matching behaviour as Java's Pattern class.
     for (int i = 0; i < 1000; i++) {
+      caseSensitiveQuery = true;      
```

Comment:
should use randomization ?

Reply:
The randomisation comes from the choice of string mutation later that produces the regex - this is just initialising the variable.